Gaijinizer patch/Map047.txt

```diff
@@ -4,109 +4,111 @@
 
 道の先になにかいるぞ。
 > CONTEXT: Map047/events/38/pages/0/8/Dialogue/yuuma
-Wait, \\N[1]. I have something to do ahead of the
-road.#MTLed
+Wait, \\N[1]. There's something ahead of you.#Ron
 > END STRING
 
 > BEGIN STRING
 な、なんだあれ…。ここじゃ後ろ姿しか見えないけど……。
 
 あれも『悪魔』なのか…？
 > CONTEXT: Map047/events/38/pages/0/15/Dialogue/kazuya
-Well, what? I can only see the back view here ...
-Is that "demon" ...?#MTLed
+W- what the hell's  that? I can only see its back,
```

Comment:
I'd remove the extra space, and keep the ellipses here.
>_**W-what the hell's that? I can only see its back...
Is it a demon...?**_

Reply:
All done.